products/db-zap-opendata/src/visible_projects.py

```diff
@@ -101,8 +101,7 @@ def make_staging_table(sql_engine, dataset_name) -> None:
                     %(source_table_name)s.dcp_userinputborough as unverified_borough,
                     %(source_table_name)s.dcp_userinputblock as unverified_block,
                     %(source_table_name)s.dcp_userinputlot as unverified_lot
-             from %(source_table_name)s INNER JOIN dcp_projects_recoded
-            on SUBSTRING(%(source_table_name)s.dcp_name, 0,10) = dcp_projects_recoded.project_id);
+            FROM %(source_table_name)s);
```

Comment:
Did you do any testing on the results of this? Main case I could see is that this is a specific filtering of the source data to only dcp recoded projects

Reply:
the final outputs (the "visible" versions of Projects and BBLs tables) are as expected based on row counts and file sizes

this filtering was redundant since [the same filtering](https://github.com/NYCPlanning/data-engineering/blob/0232a12a6f414bb321e47b4f41381b8f9849d091/products/db-zap-opendata/src/visible_projects.py#L188-L189) takes place when it's actually needed: when BBL records have to be limited to DCP projects with "General Public" visibility

Reply:
Gotcha. I'm a little more confused after looking through that whole file more, though for reasons unrelated to this PR, just the general logic in here. 

Are the `_staging` tables used at all in export? Like for our own use? Or really just temporary, or what

Reply:
`_staging` tables aren't exported. they're used to select and rename the columns we want for the subsequent `_recoded` and `_visible` tables

it goes `_crm` -> `_staing` -> `_recoded` -> `_visible`

[previously](https://github.com/NYCPlanning/db-zap-opendata/pull/45/files#diff-e6905d6f9c665e3efc78cde7015333a2048c899407361715dfbd7a40cdfd015aL28), the logic and columns were more confusing so I added the use of intermediate tables to try and better separate stages of the export